crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -116,6 +122,12 @@ fn generate_qobject_definitions(qobject_idents: &QObjectNames) -> Result<Generat
         }
     };
 
+    let base_upcast = if let Some(base) = base {
+        quote! { impl cxx_qt::Upcast<#base> for #rust_struct_name_rust {} }
```

Comment:
base here is a string rather than ident? so it's probably do `impl cxx_qt::Upcast<"T">` ?

Reply:
ah yup, forgot i'd switched back to string there